### PR TITLE
fixed Regression reuse plus apis

### PR DESCRIPTION
**Pull request by rftemple**
Signed-off-by: Templeton, Randall <randall.f.templeton@intel.com>

### CODE REVIEW

**Review by KiranSukhavasi — CHANGES_REQUESTED** [review]
(https://jira.devtools.intel.com/browse/SDOPT-9230)[Regression]

P2 https://jira.devtools.intel.com/browse/SDOPT-9235 [FDO 1.1][PRI] With PR 456 TO0 failing after message 22 using individual components

**Review by KiranSukhavasi — APPROVED** [review]

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OwnerVoucher.java [hunk 1]
```diff
@@ -4,24 +4,72 @@
...
+    try {
+
+      CriteriaBuilder cb = session.getCriteriaBuilder();
+      CriteriaQuery<OnboardingVoucher> cq = cb.createQuery(OnboardingVoucher.class);
+      Root<OnboardingVoucher> rootEntry = cq.from(OnboardingVoucher.class);
+      CriteriaQuery<OnboardingVoucher> all = cq.select(rootEntry);
+
+      TypedQuery<OnboardingVoucher> allQuery = session.createQuery(all);
+      List<OnboardingVoucher> list = allQuery.getResultList();
+      OnboardingConfig onboardConfig = new OnboardConfigSupplier().get();
```
Comment (UnitedCloudServices) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OwnerVoucher.java:52]: Variable was never read after being assigned.

**Review by UnitedCloudServices — CHANGES_REQUESTED** [review]

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OwnerVoucher.java (hunk 1 above)
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OwnerVoucher.java:52]: onboardConfig - Variable was never read after being assigned.

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/StandardMessageDispatcher.java [hunk 2]
```diff
@@ -1093,8 +1108,13 @@ protected void doSetupDevice(DispatchMessage request, DispatchMessage response)
...
     devInfoReady.setHmac(newMac);
 
-    //todo: get from config
-    devInfoReady.setMaxMessageSize(null);
+    Integer maxSvi = getWorker(MaxServiceInfoSupplier.class).get();
+    devInfoReady.setMaxMessageSize(maxSvi);
+    if (maxSvi == null) {
+      logger.info("max service info size is null (default)");
+    } else {
+      logger.info("max service info size size is " + maxSvi);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/StandardMessageDispatcher.java:1116]: Size Size

Comment (rftemple) [reply protocol/src/main/java/org/fidoalliance/fdo/protocol/StandardMessageDispatcher.java]: fixed in commit

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OwnerVoucher.java (hunk 1 above)
Comment (rftemple) [reply protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OwnerVoucher.java]: fixed in commit

**Review by KiranSukhavasi — APPROVED** [review]

**Review by himanshu7196 — APPROVED** [review]

**Pull request merged**